### DISCUSSION TITLE
extract_tables怎么将row中倾斜的文字去除

### DISCUSSION THREAD

**Discussion by zyc1128** [jsvine/pdfplumber#1154] [Ask for help with specific PDFs]
## Describe the bug

*A clear and concise description of what the bug is.*

## Have you tried [repairing](https://github.com/jsvine/pdfplumber/blob/stable/docs/repairing.md) the PDF?

*Please try running your code with `pdfplumber.open(..., repair=True)` before submitting a bug report.*

## Code to reproduce the problem

*Paste it here, or attach a Python file.*

## PDF file

*Please attach any PDFs necessary to reproduce the problem.*

*If you need to redact text in a sensitive PDF, you can run it through [JoshData/pdf-redactor](https://github.com/JoshData/pdf-redactor).*

## Expected behavior

*What did you expect the result __should__ have been?*

## Actual behavior

*What actually happened, instead?*

## Screenshots

*If applicable, add screenshots to help explain your problem.*

## Environment

- pdfplumber version: [e.g., 0.5.22]
- Python version: [e.g., 3.8.1]
- OS: [e.g., Mac, Linux, etc.]

## Additional context

*Add any other context/notes about the problem here.*

**Comment by jsvine** [comment]
Please provide more details. Thank you.
